javascript.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable camelcase */
+/* eslint-disable prefer-destructuring */
```

Comment:
- [ ] it would be better to remove the disable eslint in order to ensure professional and clean code